web/ui/react-app/package.json

```diff
@@ -19,7 +19,7 @@
     "@lezer/lr": "^1.4.2",
     "@nexucis/fuzzy": "^0.4.1",
     "@nexucis/kvsearch": "^0.8.1",
-    "@prometheus-io/codemirror-promql": "0.54.1",
+    "@prometheus-io/codemirror-promql": "0.300.0-rc.0",
```

Comment:
@jan--f that's the same issue than before. This file shouldn't be updated with the new version :)